facilitator/Dockerfile

```diff
@@ -38,10 +38,14 @@ ARG BUILD_INFO=unspecified
 ENV OPENSSL_STATIC=true
 
 # This cargo build command must match the one above, or the build cache will not be reused.
-RUN cargo build --manifest-path ./facilitator/Cargo.toml
-# We build in debug mode so the build runs quickly, then strip the binary for size.
-RUN strip facilitator/target/debug/facilitator
+RUN cargo build --manifest-path ./facilitator/Cargo.toml --release
+
+# Build a minimal container from Alpine containing only the stripped binary and
+# no intermediate build artifacts
+FROM alpine:3.14
```

Comment:
I believe once openssl is removed we can build from `scratch`?

Reply:
I don't think so, because we still need `libc` and a few other .sos for facilitator:
```
> ldd target/debug/facilitator
	linux-vdso.so.1 (0x00007fff9ab7f000)
	libssl.so.1.1 => /lib64/libssl.so.1.1 (0x00007f9515fea000)
	libcrypto.so.1.1 => /lib64/libcrypto.so.1.1 (0x00007f9515cfc000)
	libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f9515ce1000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f9515cc0000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f9515b7c000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007f9515b75000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f95159a4000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f951821e000)
	libz.so.1 => /lib64/libz.so.1 (0x00007f951598a000)
```
The `libssl` dependency should go away but the rest are still real.